src/dask_awkward/lib/core.py

```diff
@@ -978,22 +997,32 @@ def _call_behavior_property(self, property_name: str) -> Any:
         if a user follows the pattern:
 
         class SomeMixin:
-            def get_the_property(self, __dask_array__ = None):
-                ...
 
-            the_property = property(get_the_property)
+            @property
+            def the_property()
```

Comment:
```suggestion
            @property
            def the_property(self):
                ...
```

Reply:
The same thoughts apply here as https://github.com/dask-contrib/dask-awkward/pull/200/files#r1148017561